### DISCUSSION TITLE
Can you make an autoplay command for the commands?

### DISCUSSION THREAD

**Discussion by amir00am** [SudhanPlayz/Discord-MusicBot#1313] [Ideas]
All the commands are complete, but there is no useful command like autoplay

**Comment by LewdHuTao** [comment]
what version are you using right now? if you're using v4 please update to v5.x or v5. Then you can use /autoqueue command

**Reply by amir00am** [reply]
no i mean autoplay

**Reply by LewdHuTao** [reply]
> no i mean autoplay

autoqueue is autoplay.
